tests/pipeline-all-amps/pom.xml

```diff
@@ -61,7 +61,7 @@
             <id>internal</id>
             <!-- publishes "image:<acs.version>-<build-number>" on Quay -->
             <properties>
-                <image.tag>${acs.version}-${build-number}</image.tag>
+                <image.tag>${acs.branch}-${build-number}</image.tag>
```

Comment:
@alandavis can this affect other non master HEAD branch builds we're running?
using the slash in the branch name would create an invalid image tag and probably break the build

Reply:
I think we do it only for develop

Reply:
You are correct. We have to support other ACS releases. This change would switch from tags names such as

6.2.3-9876
6.2.2-4567
7.0.0-1234

To 
release/6.2.N-9876     -- note the N and /
release/6.2.2-4567
master-1234

I really don't have a view as to what the tag names are for the single pipeline, other than that I would have though what we have at the comment would have been simpler, but I'm not consuming the images.

Reply:
is it correct this job runs on master and release/* or can we just do master? that would solve the problem

https://github.com/Alfresco/acs-packaging/blob/master/.travis.yml#L38-L39
https://github.com/Alfresco/acs-packaging/blob/master/.travis.yml#L195-L198

Reply:
It runs currently on both.
If your regex is: 7\.0\.0-[0-9]+
is there a problem?

I don't know what problem you are trying to solve.